packages/@o3r/eslint-config/src/rules/typescript/eslint-typescript.cjs

```diff
@@ -88,7 +88,11 @@ const config = [
       '@typescript-eslint/no-non-null-assertion': 'off',
       '@typescript-eslint/no-parameter-properties': 'off',
       '@typescript-eslint/no-redeclare': 'error',
+      // Why do I turn it to off?
+      // If we keep it false, it should be added to the doc.
       '@typescript-eslint/no-redundant-type-constituents': 'off',
+      // Kilian wanted to set it to error.
```

Comment:
(the purpose of enforcing it to error is to prepare `mjs` migration)

Reply:
The question is more is it something we recommend to everyone or only to us ?

Reply:
To everyone, we should all be prepared to `mjs` :)